ivy/functional/frontends/torch/tensor.py

```diff
@@ -1492,6 +1495,12 @@ def tile(self, *reps):
             reps = reps[0]
         return torch_frontend.tile(self, reps)
 
+    def slice_scatter(self, updates, begin, size, axis):
+        slices = [slice(None)] * self.ndim
+        slices[axis] = slice(begin, begin + size)
+        self.ivy_array[tuple(slices)] = updates.ivy_array
+        return self
```

Comment:
Hi, the `slice_scatter` is not a In-place method so the `self.ivy_array` should not be changed.